ui/app/selectors/selectors.js

```diff
@@ -127,9 +127,16 @@ export function getMetaMaskAccountsRaw(state) {
 }
 
 export function getMetaMaskCachedBalances(state) {
+  const chainId = getCurrentChainId(state);
+
+  // Fallback to fetching cached balances from network id
```

Comment:
TIL that we persist our cached account balances. I was going to suggest that we could remove the fallback now, because they aren't persisted, but I checked and I was mistaken.